po/extra/pt_BR.po

```diff
@@ -0,0 +1,222 @@
+# Russian translation for com.github.avojak.warble
```

Comment:
Let's update the headers on both .po files to say Portuguese instead of Russian. You can also replace with your name on the last line too for proper attribution!